### ISSUE TITLE
No matching distribution found for tensorflow

### ISSUE THREAD

**Issue by KochankovID** [tensorflow/tensorflow#52768]
Used one of your Dockerfiles to build a docker image with TensorFlow installed (also I've tried to do it manually and used python docker image and latest Ubuntu image) but all the time I get same error: 

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/33292483/139205340-79248dec-c487-4589-be45-9593f0492dd8.png">

I've tried to update pip and setuptools.

**Comment by mohantym** [comment]
Hi @KochankovID ! Sorry for the late response. Did you follow instructions from [here ](https://www.tensorflow.org/install/docker) ?You can check these threads to upgrade your setup tools.[link1](https://stackoverflow.com/questions/59469537/how-can-i-upgrade-pips-setup-tools),[link2 ](https://github.com/pypa/pip/issues/8395)and confirm the version using this command  `pip --version` . Thank you!

**Comment by KochankovID** [comment]
I actually found out what my problem and hope it'll help others.

I use mac with m1 processor which has arm architecture. So, when I use Docker images (as Ubuntu or others) it downloads images compiled for aarch64 platform (kind of Linux for arm) because it's my host platform.

But looks like TensorFlow doesn't have aarch64 package on PyPi and doesn't support this platform.
And that's why I had such error. Pip can't find distribution package for aarch64 platform.

Basically, mac can run x82_64 images (but they are a bit slower). In order to fix this problem, one can just specify platform of docker image:

FROM amd64/python:3.9-bullseye

or and64/<name of your image>

@mohantym  Thank you for your response. Hope some day TensorFlow will support aarch64 platform :)

**Comment by Prem-Deep9** [comment]
Hi , I have a pip install  tensorflow command in my dockerfile and trying to build for arm arch. What do you suggest my statement needs to modified to be able to install tensorflow.
